### ISSUE TITLE
Update for HC Players?

### ISSUE THREAD

**Issue by 8EightySixed6** [CosminPOP/LFT#26]
Is there a possible way to update this so HC can group with other HC easier?

**Comment by Lexiebean** [comment]
Yeah, it's meant to be in the current version but I left a small mistake in. It'll work next patch.
